_maps/yogstation/RandomRuins/StationRuins/BoxStation/bar_xeno.dmm

```diff
@@ -0,0 +1,1655 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```

Comment:
I have never been as confident in stating a subjective opinion as fact than I am when I say that this looks terrible and is an incredibly boring concept.